.vscode/settings.json

```diff
@@ -2,5 +2,5 @@
   "editor.formatOnSave": true,
   "files.insertFinalNewline": true,
   "files.trimFinalNewlines": true,
-  "editor.tabSize": 4
+  "editor.tabSize": 2
```

Comment:
we'll have to run `format` on the whole repo for this to work in full effect I think?

Reply:
Yeah. This is already in .prettierrc. Maybe we will want to do the full change in a later PR?

Reply:
yeah wanna remove for now? might be prettier that's inconsistent b/c hackerAPI config is `tabWidth: 4` for vscode and prettier

Reply:
Do you mean to say change prettierrc to be 4? Or leave the inconsistency?

Reply:
Done